themes/poetry/layouts/shortcodes/tip.html

```diff
@@ -1,4 +0,0 @@
-<div class="admonition tip">
```

Comment:
This was not used anywhere, and there's no specific CSS for it anyway, unlike `note` or `warning` that have dedicated icons.